resources/js/components/open/forms/components/FormEditor.vue

```diff
@@ -2,11 +2,27 @@
   <div v-if="form" id="form-editor" class="w-full flex border-t flex-grow relative overflow-x-hidden">
     <!-- Form fields selection -->
     <v-tour name="tutorial" :steps="steps" />
-    <div class="w-full md:w-1/2 lg:w-2/5 border-r relative overflow-y-scroll md:max-w-sm flex-shrink-0">
+    <div class="w-full md:w-1/2 lg:w-2/5 border-r relative overflow-y-scroll md:max-w-sm flex-shrink-0 p-4">
       <div class="p-5 bg-blue-50 border-b text-nt-blue-dark md:hidden">
         We suggest you create this form on a device with a larger screen such as computed. That will allow you
         to preview your form changes.
       </div>
+      
+      <a href="javascript:;" @click="$router.back()" class="flex text-blue mb-5">
```

Comment:
what's this `href="javascript:;"` ?

Reply:
You need to use `href="#"` instead, and then use `@click.prevent`

Reply:
ok